docs/yul_language/builtins.rst

```diff
@@ -0,0 +1,202 @@
+Builtins
+========
+
+Most operations in Yul are performed via builtin functions. Solang supports
+most builtins, however memory operations and chain operations are not yet implemented.
+They will be added in a future release.
```

Comment:
This section should NOT say "they will be added", because many will never be added. We should add something about the origin of the builtins: 

yul builtins are low level EVM instructions and many are ethereum specific (link to https://ethereum.org/en/developers/docs/evm/opcodes/). On Solana or Substrate these instructions do not exist, and because of low-level differences between the chains, it is impossible to implement some builtins. For example ``delegatecall`` and ``staticcall`` just does not exist as a concept on either chain.